frontend/views/containers/chatroom/MessageBase.vue

```diff
@@ -18,57 +18,38 @@
           span.has-text-1 {{ humanDate(datetime, { hour: 'numeric', minute: 'numeric' }) }}
 
       slot(name='body')
-        p.c-replying.custom-markdown-content(
-          v-if='replyingMessage'
-          @click='$emit("reply-message-clicked")'
-        )
-          template(v-for='(objReplyMessage, index) in replyMessageObjects')
-            span(
-              v-if='isText(objReplyMessage)'
-              v-safe-html:a='objReplyMessage.text'
-            )
-            span.c-member-mention(
-              v-else-if='isMemberMention(objReplyMessage)'
-              :class='{"c-mention-to-me": objReplyMessage.toMe}'
-            ) {{ objReplyMessage.text }}
-            span.c-channel-mention(
-              v-else-if='isChannelMention(objReplyMessage)'
-              :tabindex='objReplyMessage.disabled ? undefined : 0'
-              :class='{ "is-disabled": objReplyMessage.disabled }'
-              @click='navigateToChatRoom(objReplyMessage)'
-              @keyup.enter='navigateToChatRoom(objReplyMessage)'
-            )
-              i(:class='"icon-" + objText.icon')
-              span {{ objText.text }}
+        template(v-if='replyingMessage')
+          render-message-with-markdown(
+            v-if='shouldRenderMarkdown'
+            :isReplyingMessage='true'
+            :text='replyingMessage'
+            @click='$emit("reply-message-clicked")'
+          )
+          render-message-text.c-replying(
+            v-else
+            :isReplyingMessage='true'
+            :text='replyingMessage'
+            @click='$emit("reply-message-clicked")'
+          )
```

Comment:
I think we can simplify this code using `:is` attribute just like how we do in `ChatMain.vue`.